### DISCUSSION TITLE
check the spelling for commit message

### DISCUSSION THREAD

**Discussion by gsaple** [jesseduffield/lazygit#2753] [Q&A]
just wondering how do you guys check the spelling for the commit message (summary)?

**Comment by mark2185** [comment]
Good ol' "eh, good enough".

On a more serious note, you can use any software you usually use for that and probably enable spellchecking on commits through a custom command.

**Comment by David-Else** [comment]
Use `ltex-ls` language server in Helix and `C` to commit in Lazygit, it then launches Helix and checks the commit message for spelling and grammar problems as you type. I am using Kitty as the terminal.  https://github.com/valentjn/ltex-ls

**Comment by trey** [comment]
I'd also like this. I use CSpell in Helix and even just being able to jump out to Helix w/CSpell to write my commit message would be great.

**Reply by stefanhaller** [reply]
This should be easy. Either use `C` instead of `c` to commit, this will put you right into helix from the start; or, if you used `c` and started typing in lazygit, use `ctrl-o`, `e` to jump out to helix. Or am I misunderstanding what you want?

**Reply by trey** [reply]
Holy crap, that's awesome! I totally missed that option in the keybindings list. Thank you!
